### DISCUSSION TITLE
Welcome to reviewdog Discussions!

### DISCUSSION THREAD

**Discussion by haya14busa** [reviewdog/reviewdog#834] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by haya14busa** [comment]
:)

**Reply by rickstaa** [reply]
:smiley:

**Comment by varunsh-coder** [comment]
Hi @haya14busa! I am the founder of [Step Security](https://www.stepsecurity.io), a startup to stop software supply chain attacks. I am looking for design partners to give me early feedback and be early adopters. I would love to partner with you on this. If you are interested, please email me at info@stepsecurity.io. Thanks!
